src/components/SearchGridFilter.vue

```diff
@@ -131,8 +131,9 @@ export default {
 
 .search-filters_search-by,
 .clear-filters {
-  margin-top: 0.4em;
-  margin-left: 24px;
+  margin: 24px;
```

Comment:
You should use the Vocabulary spacing classes for margins and paddings https://cc-vocabulary.netlify.com/?path=/docs/tokens-spacing--figma

Reply:
Yup Done :)